test/helpers/hosts_and_hostgroups_helper_test.rb

```diff
@@ -244,4 +260,9 @@ def setup
     @hostgroup.medium_id = id
     assert_nil kickstart_repository_id(::Host.new, :selected_host_group => @hostgroup)
   end
+
+  test 'repository_url must render the right path based on host configuration' do
+    PATH = "http://#{@host.content_source.hostname}/pulp/repos/#{@host.organization.label}/#{@host.lifecycle_environment}/custom/zoo/zoo/".freeze
```

Comment:
This is not a good case for using constant: constant is a global thing and should be used just for real constants, please use local variable instaed

Reply:
damnn, i was sure that changed it, thanks